Modules/getpath.c

```diff
@@ -873,6 +873,11 @@ _PyConfig_InitPathConfig(PyConfig *config, int compute_path_config)
         !decode_to_dict(dict, "os_name", "nt") ||
 #elif defined(__APPLE__)
         !decode_to_dict(dict, "os_name", "darwin") ||
+#ifdef WITH_NEXT_FRAMEWORK
+        !int_to_dict(dict, "WITH_NEXT_FRAMEWORK", 1) ||
+#else
+        !int_to_dict(dict, "WITH_NEXT_FRAMEWORK", 0) ||
+#endif
```

Comment:
I think I'd prefer to have the variable defined in all cases, just to avoid the risk of unhandled NameErrors in getpath.py.

Reply:
Good point. I'll move the definition to outside of the ``__APPLE__`` block.